packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx

```diff
@@ -61,6 +61,33 @@ function getLabelStyles(props: ITextFieldStyleProps): IStyleFunctionOrObject<ILa
   });
 }
 
+export const getInputFocusBorder = (
```

Comment:
I think rather than using the version from TextField everywhere, we should move this to the styling package. Importing TextField styles from controls which otherwise don't import TextField isn't ideal for bundle size reasons.